blank-canvas-blocks/sass/blocks/_code.scss

```diff
@@ -0,0 +1,4 @@
+// TODO: This can be removed when Gutenberg applies fontFamily correcly to .wp-block-code code
```

Comment:
```suggestion
// TODO: This can be removed when Gutenberg applies fontFamily correcly to .wp-block-code code (https://github.com/WordPress/gutenberg/issues/31135)
```